aten/src/ATen/native/layer_norm.cpp

```diff
@@ -340,6 +340,14 @@ Tensor rms_norm_symint(
     return std::get<0>(rms_norm_composite(input, IntArrayRef(reinterpret_cast<const int64_t*>(normalized_shape.data()), normalized_shape.size()), weight_opt, eps));
   }
 
+  if (weight_opt.has_value() && weight_opt.value().defined() && weight_opt.value().dtype() != input.dtype()) {
+    TORCH_WARN_ONCE(
+      "Mismatch dtype between input and module: input dtype = ", input.dtype(),
+      ", module dtype = ", weight_opt.value().dtype()
```

Comment:
why is it module dtype and not weight dtype?

Reply:
Just say weight dtype is wrong, this function doesn't have to be called through module, weight dtype is clear enough